roadmap/implementers-guide/src/node/approval/approval-distribution.md

```diff
@@ -182,4 +179,15 @@ Imports an approval signature referenced by block hash and candidate index.
   * Dispatch a `ApprovalDistributionV1Message::Approval(approval)` to all peers in the `BlockEntry`'s `known_by` set, excluding the peer in the `source`, if `source` has kind `MessageSource::Peer`. Add the fingerprint of the assignment to the knowledge of each peer. Note that this obeys the politeness conditions:
     * We guarantee elsewhere that all peers within `known_by` are aware of all assignments relative to the block.
     * We've checked that this specific approval has a corresponding assignment within the `BlockEntry`.
-    * Thus, all peers are aware of the assignment or have a message to them in-flight which will make them so. 
\ No newline at end of file
+    * Thus, all peers are aware of the assignment or have a message to them in-flight which will make them so. 
+
+
+#### `unify_with_peer(peer: PeerId, view)`:
+
+For each block in the view:
+  1. Initialize `fresh_blocks = {}`
```

Comment:
We should be explicit that `fresh_blocks` is a set, not a map.

Reply:
why?

Reply:
Because we're writing this for the future, and the `{}` notation is ambiguous. It's possible to figure it out from later context, but why make it harder on the reader?